### PR TITLE
add note about enterprise plans

### PR DESCRIPTION
**Pull request by mirnawong1**
add clarifying info that acct-level connections feature isn't yet available to enterprise plans (and free plans) yet. [internal slack thread](https://dbt-labs.slack.com/archives/C05CJT5MJF2/p1722613501740939)

### CODE REVIEW

File: website/docs/docs/cloud/connect-data-platform/about-connections.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ Up until July 2024, connections were nested under projects. One dbt Cloud projec
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connections-legacy-model.png" width="60%" title="Previous connection model"/>
 
-We are rolling out an important change that moves connection management to the account level. The following connection management section describes these changes. 
+We are rolling out an important change that moves connection management to the account level. This feature is being rolled out in phases over the coming weeks.  Please note that this change isn't yet available to Enterprise plans, but we anticipate it will be soon.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/connect-data-platform/about-connections.md]: @Fleid do we want to specifically say enterprise AND our free developer plans? or did you mean another free plan?

**Comment by mirnawong1** [comment]
if anyone from the docs team reviews, i wonder if the image in the callout is taking up a lot of space and overshadows the actual content? i would suggest removing the image and adding another sub header (h3 or h4 Comparing environments or Compare account-level and project-level environments (but that's so long)). cc @matthewshaver 


Screenshot 2024-08-12 at 14 55 27

File: website/docs/docs/cloud/connect-data-platform/about-connections.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/cloud/connect-data-platform/about-connections.md]: also making sure this is in general and not specific to jus terraform users?

**Review by nataliefiann — APPROVED** [review]
Hiya @mirnawong1 

Thanks for raising this. Great PR and no suggestions or recommendations on my part. I'll approve this for you. 

Kind Regards
Natalie

**Comment by mirnawong1** [comment]
Changing PR as this has been rolled out now

**Pull request merged**